docs/rholang/rholangmatchingtut.md

```diff
@@ -0,0 +1,181 @@
+# A Tutorial for Matching in Rholang
+
+## Patterns we are allowed to match
+
+Let's clarify the implications of the following sentence from the Rholang tutorial: "There are two kinds of values in Rholang, names and processes. Patterns are names or processes with free variables, which appear to the left of an arrow in a `for` or a `match`." This has a couple of implications that were not explicitly stated in the tutorial.
```

Comment:
or after the `match` keyword.

Reply:
> This has a couple of implications that were not explicitly stated in the tutorial.

This sentence is somewhat redundant with the previous

Reply:
Good catch.